lib/Backend/GlobOpt.cpp

```diff
@@ -13252,12 +13261,23 @@ GlobOpt::OptArgLenAndConst(IR::Instr* instr, Value** src1Val)
                     {
                         IR::Opnd * undefined = IR::AddrOpnd::New(instr->m_func->GetScriptContextInfo()->GetUndefinedAddr(), IR::AddrOpndKindDynamicVar, instr->m_func, true);
                         undefined->SetValueType(ValueType::Undefined);
-                        replaceInstr(undefined);
+                        replaceInstr(undefined, Js::OpCode::Ld_A);
```

Comment:
Isn't this incorrect for TypeofElem?

Reply:
Should it still be TypeOf opcode for undefined, does that not return undefined?

Reply:
the string "undefined" not the object undefined

---
In reply to: [222496035](https://github.com/Microsoft/ChakraCore/pull/5733#discussion_r222496035) [](ancestors = 222496035)

Reply:
Gotcha, I have made the modifications for that.